src/components/itemList/SearchBar.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import style from "./SearchBar.module.css";
+
+
+function SearchBar({ handleOrder }) {
+  const handleChange = (e) => {
+    handleOrder(e.target.value);
+  };
+  return (
+    <div className={style.utilBox}>
+      <input
+        className={style.searchBar}
+        placeholder="검색할 상품을 입력해주세요"
+      ></input>
+      <button className={style.btnStyle}>상품 등록하기</button>
```

Comment:
P2:
button 태그의 경우 type을 명시해주시는 것이 좋습니다~
```suggestion
      <button type='button' className={style.btnStyle}>상품 등록하기</button>
```